util/trace_replay.h

```diff
@@ -72,10 +72,12 @@ class Tracer {
   Status WriteHeader();
   Status WriteFooter();
   Status WriteTrace(const Trace& trace);
+  bool ShouldSkipTrace();
 
   Env* env_;
   TraceOptions trace_options_;
   std::unique_ptr<TraceWriter> trace_writer_;
+  uint64_t write_request_count_;
```

Comment:
Lets rename this to trace_request_count_, as we are tracing both "read" and "write" APIs.